packages/dx-react-grid-bootstrap3/src/templates/search-panel-input.test.jsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { SearchPanelInput } from './search-panel-input';
+
+let changeSearchValue;
+const getMessage = jest.fn().mockReturnValue('placeholder');
```

Comment:
Typically we define the `defaultProps` constant which is an object containing all the properties marked as required.